webclient/app/src/app/commons/appTemplateDialog/AppTemplateDialog.jsx

```diff
@@ -102,12 +137,44 @@ function AppTemplateDialog(props) {
                 autoComplete="off"
             >
                 <ErrorAlert alert={alert} onClose={closeAlert} />
-                <TextField fullWidth label={t("appTemplateDialog.location")} value={internalAppTemplate.location} name="location" onChange={handleChange} />
-                <TextField fullWidth label={t("appTemplateDialog.branch")} value={internalAppTemplate.branch} name="branch" onChange={handleChange} />
-                <TextField fullWidth label={t("appTemplateDialog.description")} value={internalAppTemplate.description} name="description" onChange={handleChange} />
+                <SimpleValidatedTextField
+                    fullWidth
+                    label={t("appTemplateDialog.location") + "*" }
+                    value={internalAppTemplate.location}
+                    name="location" 
+                    onChange={handleChange}
+                    iscreate={+isCreateDialog}
```

Comment:
`+isCreateDialog` is correct?

Reply:
otherwise the boolean value was not taken as boolean